proto/interchain_security/ccv/provider/v1/provider.proto

```diff
@@ -120,8 +120,8 @@ message ConsumerRemovalProposal {
   string title = 1;
   // the description of the proposal
   string description = 2;
-  // the chain-id of the consumer chain to be stopped
-  string chain_id = 3;
+  // the consumer id of the consumer chain to be stopped
+  string consumer_id = 3;
```

Comment:
Do we still need the `ConsumerRemovalProposal` and `ConsumerModificationProposal`? I thought the plan is to deprecate legacy gov proposals from ICS.

Reply:
No we do not. Those will be eventually removed. Probably renaming their fields was not worth it, I agree.